packages/jest/package.json

```diff
@@ -31,7 +31,8 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/facebook/jest"
+    "url": "https://github.com/facebook/jest.git",
+    "directory": "packages/jest"
```

Comment:
is this something `constraints` warned about, or just something you noticed?

Reply:
constraints warn me about this.  I add a rule to make our `repository.url` consistent. 

```prolog
gen_enforced_field(WorkspaceCwd, 'repository.url', 'https://github.com/facebook/jest.git') :-
  \+ workspace_field(WorkspaceCwd, 'private', true).
```

Reply:
nice!